src/main/java/org/elasticsearch/plugins/AbstractPlugin.java

```diff
@@ -115,7 +115,7 @@ public void processModule(Module module) {
     }
 
     @Override
-    public Settings additionalSettings() {
+    public Settings additionalSettings(Settings settings) {
         return ImmutableSettings.Builder.EMPTY_SETTINGS;
```

Comment:
Nope, because it is about the additional settings a plugin can set. This just means that by default no additional settings are added to a node's settings.